Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3_test.py

```diff
@@ -131,13 +131,41 @@ def test_get_intelligence_command(self, mocker):
         client = mock_client()
 
         # run
-        intelligence_relationships, outputs = get_intelligence(client, INDICATOR[0], FeedIndicatorType.IP)
+        intelligence_relationships, outputs = get_intelligence(client, INDICATOR[0], FeedIndicatorType.URL)
 
         # validate
         assert outputs.get('Actor')
         assert not outputs.get('Campaign')
         assert intelligence_relationships
 
+    def test_get_intelligence_with_false_create_relationships(self, mocker):
+        """
+        Given:
+            - Client with should_create_relationships=False, indicator, and indicator type
+
+        When:
+            - Call the get_intelligence command
+
+        Then:
+            - Validate that the relatiionships list does not have None values
```

Comment:
better validation: return value is a list of EntityRelationship objects (in particular, empty list if nothing is returned).